### PR TITLE
[Fix] Remove serial flag from wasm feature within synthesizer-process

### PR DESCRIPTION
**Pull request by iamalwaysuncomfortable**
## Motivation

One of the feature flags specified in the wasm flag `snarkvm-synthesizer-process` specified a `serial` flag. Unforutnately this creates a dependency cascade which ensures that the `serial` feature of snarkvm-algorithms is activated and thus Multi-Threaded `wasm` isn't possible. This would cause a significant regression in performance on `aleo.tools` and any other web property using Aleo wasm for web execution.

This PR removes the serial flag therefore re-enabling multi-threading in the wasm compilation target.

### CODE REVIEW

**Comment by iamalwaysuncomfortable** [comment]
@howardwu @ljedrz @raychu86 - hate to ask again, but can this get a rubber stamp and subsequent release? It's another blocker.

File: synthesizer/process/Cargo.toml [hunk 1]
```diff
@@ -38,7 +38,6 @@ serial = [
 wasm = [
   "console/wasm",
   "ledger-block/wasm",
-  "ledger-query/serial",
```
Comment (howardwu) [inline synthesizer/process/Cargo.toml:41]: That should read `ledger-query/wasm`

Comment (iamalwaysuncomfortable) [reply synthesizer/process/Cargo.toml:41]: @howardwu change made

**Comment by iamalwaysuncomfortable** [comment]
Updated the dependency and verified it enables multi-threading, possible to merge this?

**Review by howardwu — APPROVED** [review]
LGTM

**Pull request merged**